x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/quick_create_job.ts

```diff
@@ -330,4 +338,35 @@ export class QuickJobCreator {
 
     return mergedQueries;
   }
+
+  private async createDashboardLink(dashboard: Dashboard, datafeedConfig: estypes.MlDatafeed) {
+    if (dashboard === undefined) {
+      // embeddable may have not been in a dashboard
+      return null;
+    }
+
+    const params: DashboardAppLocatorParams = {
+      dashboardId: dashboard.id,
+      timeRange: {
+        from: '$earliest$',
+        to: '$latest$',
+        mode: 'absolute',
+      },
+      filters: getFiltersForDSLQuery(
```

Comment:
Is it worth giving this filter a user friendly label, rather than defaulting to the query string?

<img width="761" alt="image" src="https://user-images.githubusercontent.com/7405507/194086033-b7b2a3c6-6597-45aa-988a-8c19b88c75d2.png">

Reply:
Updated in https://github.com/elastic/kibana/pull/142139/commits/53b8760d830ed0a3d79f7fd441a4de8bad284216